app/viewModels/drafts/AllDraftDeparturesViewModel.scala

```diff
@@ -69,6 +77,40 @@ case class AllDraftDeparturesViewModel(items: DeparturesSummary,
 
   def pageNumber: Int = paginationViewModel.pageNumber
 
+  def sortLrn: String = sortParams match {
+    case SortByLRNAsc  => "ascending"
+    case SortByLRNDesc => "descending"
+    case _             => "none"
+  }
+
+  def sortCreatedAt: String = sortParams match {
+    case SortByCreatedAtAsc  => "ascending"
+    case SortByCreatedAtDesc => "descending"
+    case _                   => "none"
+  }
+
+  def sortLRNHref(): Call = {
+    val currentLRNState = sortLrn
+    val callParams = currentLRNState match {
+      case "descending" => Some("lrn.asc")
+      case "ascending"  => Some("lrn.dsc")
+      case _            => Some("lrn.asc")
+    }
+
+    routes.DashboardController.onPageLoad(None, lrn = lrn, callParams)
+
+  }
+
+  def sortCreatedAtHref(): Call = {
+    val currentCreatedAtState = sortCreatedAt
+    val callParams = currentCreatedAtState match {
+      case "descending" => Some("createdAt.asc")
+      case "ascending"  => Some("createdAt.dsc")
+      case _            => Some("createdAt.dsc")
+    }
```

Comment:
Again, this toggle functionality feels like it could exist in the trait, and there'd be no need for duplication